applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -280,6 +317,44 @@ def _CreateMetricsProcess(self):
                             self.model_part,
                             current_metric_variable,
                             hessian_parameters))
+        elif (self.strategy == "superconvergent_patch_recovery"):
+            assert structural_dependencies
```

Comment:
hm I like a proper error message more

Btw if you import StructMech only here then you will get a proper error message

Reply:
I prefer to use the assert, but I raise an error